src/dicom-file.c

```diff
@@ -1224,12 +1547,11 @@ static void destroy_pixel_description(struct PixelDescription *desc)
 
 
 DcmBOT *dcm_file_build_bot(DcmError **error,
-                           const DcmFile *file, const DcmDataSet *metadata)
+                           DcmFile *file, DcmDataSet *metadata)
```

Comment:
Why not keep the `const`? Wouldn't it be nicer to indicate that these arguments are not modified by the function?

Reply:
I agree, let's add a lot more const everywhere. How about adding a note to the TODO and doing them all in a pass near the end?